docs/coordinator.md

```diff
@@ -0,0 +1,50 @@
+# Coordinator
+
+## Introduction
+
+The `coordinator` is the component in charge of managing a cluster of load-test
+agents and figure out what's the maximum amount of concurrently active users
+the target Mattermost instance can support. This is achieved through the use of a simple feedback loop.
+
+When a load-test starts, the `coordinator` will begin to gradually increase the number of
+active users. At the same time it will monitor performance by querying a
+[Prometheus](https://prometheus.io/docs/introduction/overview/) server collecting metrics from the target instance.
+
+When signs of performance degradation are detected, the `coordinator` will start
+decreasing the number of active users. When overall performance goes back to an
+accepted value (results of queries fall under the configured thresholds) the `coordinator` will start once again to increase the amount
+of active users for the target instance.
+
+This process will continue until an equilibrium point is found which will
```

Comment:
How do we determine the `equilibrium point`?

Reply:
That's a good point. I've tried to give a high level view of how the coordinator works.  
What you are asking for is a bit more technical and the real answer is "we don't know yet".   
The current implementation is a bit *dumb* and will set `supportedUsers` to be the number of active users we were able to run without receiving any performance degradation events for some pre-defined amount of time.